src/mame/layout/segaai.lay

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0"?>
+<mamelayout version="2">
+	<element name="touchpad">
+<!--
+		<image file="touchpad.png" />
+-->
+	</element>
```

Comment:
Should you at least draw an inset square or something so people can see that there’s a clickable target?

Reply:
Fixed